src/test/java/com/es/phoneshop/model/product/ArrayListProductDaoTest.java

```diff
@@ -2,20 +2,57 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
 
-import static org.junit.Assert.assertTrue;
+import java.math.BigDecimal;
+import java.util.Currency;
+import java.util.List;
+
+import static org.junit.Assert.*;
 
 public class ArrayListProductDaoTest
```

Comment:
Please add more test cases to cover all scenarios and rename test methods to make them more obvious:
for example "shouldReturnProductIfPresent", "shouldThrowExceptionIfProductNotFound"